src/providers/AppState.js

```diff
@@ -44,7 +45,7 @@ function AppStateProvider({ children }) {
   }, [totalSupply, totalStaked, minThresholdStakePercentage])
 
   const balancesLoading = vaultBalance.eq(-1) || totalSupply.eq(-1)
-  const appLoading = !convictionVoting || balancesLoading || !effectiveSupply
+  const appLoading = errorFetchingApp || errorFetchingOrg ? false : !convictionVoting || balancesLoading || !effectiveSupply
```

Comment:
As commented above, we could refactor this a little bit to be:

```suggestion
  const errors =  errorFetchingApp || errorFetchingOrg
  const appLoading =  !errors && !convictionVoting || balancesLoading || !effectiveSupply
```